python/cudf/cudf/core/indexed_frame.py

```diff
@@ -3465,14 +3465,23 @@ def _binaryop(
         **kwargs,
     ):
         reflect, op = self._check_reflected_op(op)
-        operands, out_index = self._make_operands_and_index_for_binop(
+        (
+            operands,
+            out_index,
+            can_use_self_column_name,
+        ) = self._make_operands_and_index_for_binop(
             other, op, fill_value, reflect, can_reindex
         )
         if operands is NotImplemented:
             return NotImplemented
 
         return self._from_data(
-            ColumnAccessor(type(self)._colwise_binop(operands, op)),
+            ColumnAccessor(
+                type(self)._colwise_binop(operands, op),
+                level_names=None
```

Comment:
Same as in the other PR -- would prefer to see `level_names` defined before the `return` statement.

Reply:
Yup, I've made this change in the other PR. I'll merge #13772 before this one and bring those changes into this PR.